build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -235,7 +236,17 @@ class RuntimeTestConfig {
         },
       ];
 
-      this.browserify.transform = [['babelify', {plugins: [plugin]}]];
+      this.browserify.transform = [
+        [
+          'babelify',
+          {
+            // Transform "node_modules/", but ignore devDependencies.
+            'global': true,
+            'ignore': devDependencies(),
+            'plugins': [plugin],
+          },
+        ],
+      ];
```

Comment:
I was just hacking this to modify instead of overwrite the config. WDYT?

```js
const babelify = this.browserify.transform.find(t => t[0] === 'babelify');
const babelifyConfig = babelify[1];
if (babelifyConfig.plugins) {
  babelifyConfig.plugins.push(plugin);
} else {
  babelifyConfig.plugins = [plugin];
}
```

Reply:
Modifying the config seems a better option than rewriting it entirely. I wish there were a more elegant way of doing so. Okay to merge this and punt the change you propose to a separate PR?

Reply:
I like it

Reply:
Or punt, either way is fine

Reply:
Sent out #23015